project1/styles.css

```diff
@@ -0,0 +1,85 @@
+body {
+    color: rgb(85, 85, 85);
+    background-color: rgb(230, 230, 230);
+    font-family: 'Cormorant Garamond', serif; 
+    
+ 
+     
+}
+
+.wrapper {
+    margin-right: 10%;
+    margin-left: 7%;
+    padding-right: 50px;
+    padding-left: 50px;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    width: 80%; 
+    
+}
+
+h1 {
+    color: rgba(43, 65, 105, 0.87);
+    font-family: 'Lexend Mega', sans-serif;
+    font-size: 600%;
+    text-align: center; 
+    line-height: 2;
+    text-transform: uppercase;
+    margin-bottom: 20px;
+}
+
+h2 {
+    color: rgb(102, 102, 102);
+    text-align: center;
+    font-size: 23px; 
+    line-height: 0.5;
```

Comment:
I recommend not using very tiny `line-height` like this because here, the line height is actually smaller than the text, which you can see via dev tools:
![Screen Shot 2019-09-26 at 9 51 25 AM](https://user-images.githubusercontent.com/2653980/65694171-685f0880-e043-11e9-91e9-c03ccb8e4f89.png)

The blue background shows the size of the element, as a result of your `line-height`, and you can see how the text is escaping from it. This isn't causing problems now, but it will cause problems:
1. When we work on responsive styles (where your design needs to adapt for mobile, tablet and desktop situations)
2. If the text was longer and involved line wraps

As you can see, as soon as the line wraps, the text gets very jumbled:
![Screen Shot 2019-09-26 at 9 54 44 AM](https://user-images.githubusercontent.com/2653980/65694483-f0451280-e043-11e9-9729-e53e774e2401.png)